doc/manual/spec.md

```diff
@@ -86,65 +88,75 @@ definition:
+These are useful for handling pre- and post-release versions, such as
+1.0~rc1 and 2.0^a.
 
-"URL:" is a place to put a URL for more information and/or documentation
-on the software contained in the package.
+#### Release
 
-Packager: tag is meant to contain the name and email
-address of the person who "maintains" the RPM package (which may be
-different from the person who actually maintains the program the
-package contains).
+Package release, used for distinguishing between different builds
+of the same software version.
 
-### BuildArchitectures: Tag
+See Version for allowed characters and modifiers.
 
-This tag specifies the architecture which the resulting binary package
-will run on.  Typically this is a CPU architecture like sparc,
-i386. The string 'noarch' is reserved for specifying that the
-resulting binary package is platform independent.  Typical platform
-independent packages are html, perl, python, java, and ps packages.
+#### Epoch
 
-### Dependencies
+Optional numerical value which can be used to override normal version-release
+sorting order. It's use should be avoided if at all possible.
 
-#### Fine Adjustment of Automatic Dependencies
+Non-existent epoch is exactly equal to zero epoch in all version comparisons.
 
-Rpm currently supports separate "Autoreq:" and "Autoprov:" tags in a
-spec file to independently control the running of find-requires and
-find-provides. A common problem occurs when packaging a large third
-party binary which has interfaces to other third party libraries you
-do not own.  RPM will require all the third party libraries be
-installed on the target machine even though their intended use was
-optional. To rectify the situation you may turn off requirements when
-building the package by putting
+#### License
 
+Short (< 70 characters) summary of the package license. For example:
 ```
-	Autoreq: 0 
+	License: GPLv3
 ```
 
-in your spec file. Any and all requirements should be added manually using the
+#### Group
 
+Optional, short (< 70 characters) group of the package.
 ```
-	Requires: depend1, ..., dependN
+	Group: Development/Libraries
 ```
 
-in this case.
+#### Summary
+
+Short (< 70 characters) summary of the package.  
+```
+	Summary: Utility for converting mumbles into giggles
+```
 
-Similarly there is an Autoprov tag to turn off the automatic provision
-generation and a Autoreqprov to turn off both the automatic provides and
-the automatic requires generation.
+#### Source
 
-### NoSource: Tag
+Used to declare source(s) used to build the package. All sources will
+will be packaged into source rpms.
+Arbitrary number of sources may be declared, for example:
+
+```
+	Source0: mysoft-1.0.tar.gz
+	Source1: mysoft-data-1.0.zip
+```
+
+#### Patch
+
+Used to declare patches applied on top of sources. All patches declared
+will be packaged into source rpms.
+
+#### Icon
+
+Used to attach an icon to an rpm package file. Obsolete.
+
+#### Nosource
+#### Nopatch
```

Comment:
These should be "cased" correctly as `NoSource` and `NoPatch` (also, we have a `NoPatch`?!?)